pkg/lib/bundle/validate_test.go

```diff
@@ -5,52 +5,10 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/operator-framework/operator-registry/pkg/containertools/containertoolsfakes"
-
 	"github.com/sirupsen/logrus"
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestPullBundle(t *testing.T) {
```

Comment:
Why are we deleting these tests?

Reply:
See nick's comments https://github.com/operator-framework/operator-registry/pull/252#discussion_r425141054